transports/http/src/main/java/org/mule/transport/http/components/StaticResourceMessageProcessor.java

```diff
@@ -61,7 +62,24 @@ public MuleEvent process(MuleEvent event) throws MuleException
         }
 
         String path = event.getMessage().getInboundProperty(HttpConnector.HTTP_REQUEST_PATH_PROPERTY);
-        String contextPath = event.getMessage().getInboundProperty(HttpConnector.HTTP_CONTEXT_PATH_PROPERTY);
+        String contextPath = event.getMessage().getInboundProperty(org.mule.module.http.api.HttpConstants.RequestProperties.HTTP_LISTENER_PATH);
```

Comment:
Yes Dan, we do. Now I check first for the http listener property, and if it doesn't exists I use the property from the old transport.

There's a test case for the transport functionality validating that it continues to work as before.